doc/guides/modules.rst

```diff
@@ -253,9 +263,43 @@ Where 23 in the example above indicated the pull request number submitted to the
 *my_module* repository. Once the module changes are reviewed and merged, the
 revision needs to be changed to the commit hash from the module repository.
 
-
+Upmerging forks
```

Comment:
"upmerging" is an NCS term I would like to avoid in official upstream docs.

"Synchronizing forks" perhaps? Here and elsewhere in this section.

Reply:
agree, I do not really know what upmerging means in this context.